fluentui_topappbars/src/main/java/com/microsoft/fluentui/tokenized/AppBar.kt

```diff
@@ -95,7 +97,8 @@ fun AppBar(
     bottomBorder: Boolean = true,
     appTitleDelta: Float = 1.0F,
     accessoryDelta: Float = 1.0F,
-    appBarTokens: AppBarTokens? = null
+    appBarTokens: AppBarTokens? = null,
+    titleAlignment: TitleAlignment = TitleAlignment.Start
```

Comment:
please move titlealignment above the token params

Reply:
this has been removed